Source/WebService/Blackjack.WebService/ClientApp/src/app/types/gamer.ts

```diff
@@ -0,0 +1,88 @@
+export abstract class Gamer {
+    /**
+     * Bool flag indicates if the player score exceeds 21
+     */
+    public hasBusted: boolean = false;
```

Comment:
The past tense of this feels, like.. weird to me.

The name is also context-specific. Which isn't a bad thing but like, think if I was a dev coming in here who didn't know anything about blackjack but had to maintain this code, what does it mean, "to bust"?

The comment is perfect b/c it explains this, but the variable could explain it too if it were titled something like, "scoreIsMoreThan21" or something like that.

This isn't a big deal I don't think, just throwing my thoughts and perspective out there for you to take in and consider.

Reply:
I felt like leaving this because I can't think of another way to say it. Going off your other comment on saying isActive or isEnabled I could work but for me isBusted or other present tense feels weirder.

Reply:
Yee that's fine. This was more to give my perspective and thoughts that I had as my internal monologue was determining how to give this concept a name.

That's why I was sayin this isn't a big deal and can stay how it is. Just wanted to put the mental gymnastics out there so you too have that perspective the next time you find yourself naming something.